$NetBSD: patch-jdk_make_com_sun_jmx_Makefile,v 1.3 2017/08/14 00:02:16 ryoon Exp $

Disable SunOS multiarch.
Let the build find intermediate libs when it wants to run intermediate rmic.

--- jdk/make/com/sun/jmx/Makefile.orig	2017-08-13 05:55:12.000000000 +0000
+++ jdk/make/com/sun/jmx/Makefile
@@ -106,7 +106,7 @@ FILES_ties   = $(subst javax/management/
 
 RMIC_JAVA = $(OUTPUTDIR)/bin/java
 # need to treat 64bit solaris differently
-ifeq ($(PLATFORM)-$(LIBARCH), solaris-amd64)
+ifeq ($(PLATFORM)-$(LIBARCH), NOMULTIARCHsolaris-amd64)
 RMIC_JAVA = $(OUTPUTDIR)/bin/amd64/java
 endif
 ifeq ($(PLATFORM)-$(LIBARCH), solaris-sparcv9)
@@ -118,7 +118,7 @@ RMIC_VM = $(RMIC_JAVA)
 else
 RMIC_VM = $(BOOT_JAVA_CMD)
 endif  
-RMIC = $(RMIC_VM) $(JAVA_TOOLS_FLAGS) -cp $(OUTPUTDIR)/classes sun.rmi.rmic.Main
+RMIC = env LD_LIBRARY_PATH=$(OUTPUTDIR)/lib/$(LIBARCH):$(OUTPUTDIR)/lib/$(LIBARCH)/jli:$(LD_LIBRARY_PATH) $(RMIC_VM) $(JAVA_TOOLS_FLAGS) -cp $(OUTPUTDIR)/classes sun.rmi.rmic.Main
 
 $(CLASSDESTDIR)/%_Stub.class: $(CLASSDESTDIR)/%.class
 	$(prep-target)
